CoreTableView/test/unit/src/au/gov/asd/tac/constellation/views/tableview/components/ExportMenuNGTest.java

```diff
@@ -207,35 +210,32 @@ public void createExportButtons() throws InterruptedException, PluginException,
         }
 
     }
-    
+
     @Test
     public void exportFileChooser() throws IOException {
         final String fileChooserTitle = "File Chooser Title";
-        final String expectedFileExtension = ".json";
-        final String fileChooserDescription = "File Filter Description";
-        
-        ExportMenuItemActionHandler handler = exportMenu.new ExportMenuItemActionHandler(fileChooserTitle, expectedFileExtension, fileChooserDescription, null);
-        
+        final FileNameExtensionFilter expectedFileExtensionFilter = FileChooser.JSON_FILE_FILTER;
+        final String fileChooserDescription = expectedFileExtensionFilter.getDescription();
+
+        ExportMenuItemActionHandler handler = exportMenu.new ExportMenuItemActionHandler(fileChooserTitle, expectedFileExtensionFilter, null);
+
         final JFileChooser fileChooser = handler.getExportFileChooser().createFileChooser();
-        
+
         assertEquals(fileChooser.getDialogTitle(), fileChooserTitle);
-        assertEquals(fileChooser.getChoosableFileFilters().length, 2);
-        
-        assertEquals(fileChooser.getChoosableFileFilters()[0].getDescription(), "All Files");
-        
-        assertEquals(fileChooser.getChoosableFileFilters()[1].getDescription(), fileChooserDescription);
-        
+        assertEquals(fileChooser.getChoosableFileFilters().length, 1);
+        assertEquals(fileChooser.getChoosableFileFilters()[0].getDescription(), fileChooserDescription);
+
         // File does not end with correct extension
         final File tmpFileInvalid = File.createTempFile("test", ".random");
-        assertEquals(fileChooser.getChoosableFileFilters()[1].accept(tmpFileInvalid), false);
-        
-        // File does not exist
-        assertEquals(fileChooser.getChoosableFileFilters()[1].accept(new File("/tmp/test" + expectedFileExtension)), false);
-        
+        assertEquals(fileChooser.getChoosableFileFilters()[0].accept(tmpFileInvalid), false);
+
         // File is valid. Exists and ends with correct extension
-        final File tmpFileValid = File.createTempFile("test", expectedFileExtension);
-        assertEquals(fileChooser.getChoosableFileFilters()[1].accept(tmpFileValid), true);
-        
+        final File tmpFileValid = File.createTempFile("test", FileExtensionConstants.JSON);
+        assertEquals(fileChooser.getChoosableFileFilters()[0].accept(tmpFileValid), true);
+
+        // Cannot create "non-existent" file for this assert
+//        // File does not exist
+//        assertEquals(fileChooser.getChoosableFileFilters()[0].accept(new File("/tmp/test" + FileExtensionConstants.JSON)), false);
```

Comment:
I presume this was passing previously so why would it not be passing now?

Reply:
I'm not sure why its not working either.

Reply:
I cannot discern why this would not be working now at all.